sources/packages/forms/src/form-definitions/studentexceptions.json

```diff
@@ -1947,8 +1944,49 @@
           },
           "properties": {},
           "customConditional": "show = data.exceptionNames.indexOf(component.key) !== -1;",
+          "lockKey": true
+        },
+        {
+          "clearOnHide": false,
+          "key": "currentYearPartnerIncomeApplicationException",
+          "input": false,
+          "components": [
+            {
+              "key": "currentYearPartnerIncomeApplicationExceptionCurrentYearPartnerIncome",
+              "label": "Current Year Partner Income",
```

Comment:
Only the last two exceptions in this list seem to be using a different case.
This one should be `Current year partner's income` probably.
Not a blocker if it is following business requirements.

![image](https://github.com/user-attachments/assets/37309f51-3c90-4d45-b41f-b1141324302e)

Reply:
I missed the content from AC. Updated it. 
![image](https://github.com/user-attachments/assets/a147db9d-71bb-4f23-902d-2aa88975f4d1)